packs/pfs-season-4-bestiary/urdefhan-high-tormentor-pfs-4-11.json

```diff
@@ -1655,7 +1655,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> The urdefhan lets out a battle cry, sending itself and its allies into a fanatical frenzy. Each ally that hears the call gains a +4 status bonus to attack rolls and damage rolls, a +2 status bonus to saving throws, and takes a -2 status penalty to AC. Affected allies must use at least one of their actions to Strike each round, if they are able (even if it means attacking an ally, object, or thin air).</p>\n<p>This lasts for [[/br 2d4 #rounds]]{2d4 rounds}.</p>\n<p>@UUID[Compendium.pf2e.bestiary-effects.Item.Effect: Stoke the Fervent - High Tormentor]{Effect: Stoke the Fervent}</p>"
+                    "value": "<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> The urdefhan lets out a battle cry, sending itself and its allies into a fanatical frenzy. Each ally that hears the call gains a +4 status bonus to attack rolls and damage rolls, a +2 status bonus to saving throws, and takes a -2 status penalty to AC. Affected allies must use at least one of their actions to Strike each round, if they are able (even if it means attacking an ally, object, or thin air).</p>\n<p>This lasts for [[/br 2d4 #rounds]]{2d4 rounds}.</p>\n<p>@UUID[Compendium.pf2e.bestiary-effects.Item.Effect: Stoke the Fervent (High Tormentor)]</p>"
```

Comment:
This is doable with AdjustModifiers rather than having a specific effect. Add it to the effect condense list for later I guess.

Reply:
Since this is an effect that's placed on allies, I think we'd need to use ActiveEffectLike to set the modifiers?